src/content/en/2021/css.md

```diff
@@ -1,10 +1,10 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: CSS
-description: TODO
-authors: []
-reviewers: []
-analysts: []
+description: CSS chapter of the 2021 Web Almanac covering trends, changes, and patterns in CSS use across the web.
+authors: [meyerweb, GeekBoySupreme]
+reviewers: [svgeesus, j9t, estelle, bkardell, argyleink, LeaVerou]
+analysts: [rviscomi, thecraftysoul]
```

Comment:
I don't recall thecraftysoul contributing, do you?

```suggestion
analysts: [rviscomi]
```

Reply:
I don’t.  I thought maybe they’d done back-of-house work with you, and I didn’t want to leave anyone out.